Content.Client/Entry/EntryPoint.cs

```diff
@@ -166,6 +167,8 @@ public override void PostInit()
             ContentContexts.SetupContexts(inputMan.Contexts);
 
             IoCManager.Resolve<IGameHud>().Initialize();
+            IoCManager.Register<DistortionMapOverlay>();
+            IoCManager.BuildGraph();
```

Comment:
It does need to be in IoC (lest it be `static`), but it **must** come after `PrototypeManager` initializes, since its shader comes from a `ShaderPrototype`.